### ISSUE TITLE
What does this module do?

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/mod-guildhouse-system#3]
So this module allows you to buy 1 zone for your guild to teleport from a list of unaccessible zones.

Once all the zones have been bought, no more guild houses :rofl:

A zone can be sold but only manually.

That's all lol

**Comment by Helias** [comment]
We can add a FAQ section in the README and write in a formal way what you wrote here ;-)
could be useful

**Comment by BarbzYHOOL** [comment]
yes but if you read carefully, you realize that it's not very engaging, i'm explaining what is good and what is bad about the module (incomplete to my eyes), to avoid people wasting time testing it
